### ISSUE TITLE
i18n - Empty solution seeds

### ISSUE THREAD

**Issue by nhcarrigan** [freeCodeCamp/freeCodeCamp#41345]
An empty solution seed with the following format does not parse correctly in Crowdin:

![image](https://user-images.githubusercontent.com/63889819/109839401-eb529e00-7bfb-11eb-8916-a58f95eca3c2.png)

Adding an empty new-line between the backtick lines *will* parse correctly in Crowdin:

![image](https://user-images.githubusercontent.com/63889819/109839454-f9082380-7bfb-11eb-870c-c0f0bc4ea0b2.png)

The following files have empty seeds with the first format and need a blank line added:

**JavaScript / Basic JavaScript**
- comment your javascript code
- declare javascript variables
- declare string variables
- initializing variables with the assignment operator
- passing values to functions with arguments
- return a value from a function with return
- write reusable javascript with functions

**JavaScript / ES6**
- create a javascript promise

**JavaScript / Object Oriented Programming**
- define a constructor function

**Front End Libraries / Bootstrap**
- Create a bootstrap headline

**Front End Libraries / React**
- Create a complex jsx element

**Quality Assurance Projects**
- metric imperial converter

**Coding Interview Prep / Data Structures**
- Create an ES6 Javascript Map

**Comment by werdna521** [comment]
Hi, I'll add those blank lines. It'll be my first time contributing to an open-source project, though.

**Comment by werdna521** [comment]
@RandellDawson Alright. I'll go through the documentation.

**Comment by CoolBlueDesigns** [comment]
Hi. First time contributor here. I have forked the free code camp files to my github page. I have cloned them locally, opened the appropriate folders into my VSCode editor and made the changes (though I was not able to find a # --seed-- in Quality Assurance Projects: metric imperial conversion) but now I have no idea of how to get the code back here. Is there some documentation with instructions on how to do that? Thanks in advance.
